.github/workflows/build_demos.yml

```diff
@@ -0,0 +1,50 @@
+name: build_demos.sh
```

Comment:
We can call workflow as `Demos`, while job is `Build`.
It will be seen in precommit checks as `Demos / Build`

If we have some tests, we can create one more stage `Tests`:
- Build stage upload artifacts
- Test downloads artifacts and perform testing.

Reply:
Renamed.
We have tests. Setting them up requires some caching to speed up, but GitHub Actions makes a full run for the first time per PR. I won't add tests here